### ISSUE TITLE
renderSync => render

### ISSUE THREAD

**Issue by chriseppstein** [joliss/broccoli-sass#58]
As I understand it, broccoli prefers synchronous file i/o for performance reasons. But I don't think that broccoli should use node-sass's renderSync method. Instead, I think it should use the async render method. We need to run several hundred sass compiles per build (number of sass files times the number of locale's we support) and this should help with throughput.

@stefanpenner tells me this is very doable, but I have forgot the specifics.

If there's no objection, I can work on a patch with a few code pointers.

**Comment by joliss** [comment]
Yeah that's fine and definitely possible - just return a promise from `updateCache`. A patch would be awesome. Can you post before/after timings as well?

As for style, you might know this already but `RSVP.denodeify` often helps with making callback-based code look pretty. :)

**Comment by chriseppstein** [comment]
> Can you post before/after timings as well?

timings of what? The tests?

**Comment by joliss** [comment]
No such "tests" ;-)

I mean timing of some non-trivial app that you have, that you're trying to speed up. (The sole motivation for this is performance, correct?)

**Comment by stefanpenner** [comment]
No the motivation is to enable sass plugins and ecosystem to have async

**Comment by joliss** [comment]
Oh, OK, that sounds great. Never mind then, no timings needed.
